.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: python
 sudo: false
 python:
-    - "2.7"
```

Comment:
Why are we removing 2.7?

Reply:
Travis job gets failed because Python2 removed from Travis CI.